rotbot.php

```diff
@@ -291,7 +291,7 @@ private function checkImage(array $picture): Image {
                 if(substr(strtolower($picture['title']),-4) == ".jpg" OR substr(strtolower($picture['title']),-5) == ".jpeg") { $image->filetype = "jpg"; }
                 else if(substr(strtolower($picture['title']),-4) == ".png") { $image->filetype = "png"; }
                 else if(substr(strtolower($picture['title']),-4) == ".gif") { $image->filetype = "gif"; }
-                // if(substr(strtolower($picture['title']),-5) == ".tiff" OR substr(strtolower($picture['title']),-4) == ".tif") { $image->filetype = "tif"; }
+                if(preg_match('/\.tiff?$/i',$picture['title'])) { $image->filetype = "tif"; }
```

Comment:
I’d keep the code consistent: either `preg_match()` for all file types or `substr(strtolower())` for all file types.

Reply:
I agree with keeping the change as small as possible to make it easier to revert. However, there are two ways to ensure consistency, and only one of them makes the diff larger – the other one actually makes it even smaller and even more focused.